src/content/blog/2025-05-05-celebrating-ten-years-of-discover.mdx

```diff
@@ -0,0 +1,68 @@
+
+## So, where did Discover come from?
+
+Before we implemented the current Discover service, UGRC utilized a platform called [ArcGIS Image Server]](https://enterprise.arcgis.com/en/image/latest/get-started/windows/what-is-arcgis-image-server-.htm). It was available for public and private use, but connections made to the service created a heavy digital load on the ArcGIS Server. This resulted in a number of problems for both UGRC and our user base; the services experienced rendering latenecy and occasional downtime. While this old service did provide significant customization options for imagery and topographic maps, it was not the best solution.
+
+For a period of time, UGRC resolved Utah's need for high-resolution aerial imagery through contracts with federal programs. These programs provided a patchwork of imagery every couple of years, but in 2012, those programs came to an end. Thus, we needed a new solution.
+
+## From Giza to Discover
+
+Shortly after high resolution aerial imagery through federal programs was no longer available, UGRC began working with Google and [Sanborn](https://sanborn.com/) to create a tile pyrimid server appliance called Giza, which serves Google imagery and basemaps as WMS and WMTS. UGRC was one of the earliest adopters of this new Google imagery program. We developed a Utah-specific version of Giza and named it Discover. In 2015, Discover officially launched as Utah's premier source for aerial imagery and basemaps services.
+
+But, as times change, so do the relationships between government entities and private corporations. After a few years of working with Google, we eventually switched to working with [Hexagon](https://hexagon.com/products/aerial-imagery) in 2018. Hexagon provides a statewide high resolution version of the federally administered [National Agriculture Imaging Program (NAIP)](https://naip-usdaonline.hub.arcgis.com/), creates tile pyramids from the imagery, and uploads it to the cloud. From there, our automated service pulls from that cloud storage to provide you with readily available aerial imagery. Since the flights are flown as part of the NAIP, updated imagery is available every three years.
+
+## Challenges to Overcome
+
+UGRC's new service worked the first time without a hitch and nothing remotely bad ever happened. There were so few bugs in the system, it may as well have been the pristine, barren landscape of Antarctica. There wasn't even one user complaint…said no developer ever, right? Like any new technology, there were more than a few hurdles to jump over in order to create a streamlined, modernized server for Utah. Many of those hurdles had to do with two unique aspects of Discover: serving the millions of pre-cached cloud-based tiles, and user access.
```

Comment:
In the last sentence we call out two problems: tile issues and user access. However, we never mention user access again. Are we missing a paragraph, or should we remove the reference to user access?

Reply:
This originally stated something different. I'll edit it to just be a 'millions of tiles' issue.